l10n_be_cooperator/report/tax_shelter_subscription_report.xml

```diff
@@ -1,205 +1,56 @@
-                            ajoutée, est inférieur ou égal à 700.000 euros ;
-                        </li>
-                        <li>la moyenne des travailleurs occupés pendant l'année
-                            est inférieure ou égale à 10.
-                        </li>
-                    </ul>
-                </p>
-
-                <p>
-                    Le maintien de la réduction d'impôt n'est possible que si
-                    l'investisseur conserve les parts de la coopérative pendant
-                    4 ans au minimum, sauf si la sortie est imposée par des
-                    conditions extérieures (telles que la faillite, par
-                    exemple). En cas de sortie volontaire avant la période de 4
-                    ans, l'avantage fiscal devra être remboursé au prorata du
-                    nombre de mois entre la sortie et les 4 ans. La coopérative
-                    s'engage à fournir pour chacune des quatre années suivant
-                    l'année d'acquisition une attestation certifiant que ces
-                    parts sont toujours en possession du souscripteur.
-                </p>
-
-                <p>
-                    L'apport de capitaux par le chef d'entreprise lui-même ou
-                    par des administrateurs existants de la société ne permet
-                    pas de bénéficier du tax shelter.
-                </p>
-
-                <p>
-                    Pour bénéficier de la réduction d'impôts, vous pouvez
-                    joindre cette lettre à votre déclaration fiscale pour les
-                    revenus <span t-field="o.declaration_id.fiscal_year" />.
-                </p>
-
-                <p>
-                    Pour toute information complémentaire, consultez le texte de
-                    l'Arrêté Royal.
-                </p>
-
-                <p>
-                    Pour le Conseil d'administration de
-                    <span t-field="o.company_id.name" />,
-                    <br />
-                    <strong t-field="o.company_id.board_representative" />
-                    <br />
-                    <img
-                        t-if="o.company_id.signature_scan"
-                        t-attf-class="mt16"
-                        t-attf-style="{{ 'min-height: 50px; max-height: 100px' if report_type == 'pdf' else '' }}"
-                        t-att-src="'data:image/png;base64,%s' % o.company_id.signature_scan.decode()"
-                    />
-                </p>
-            </div>
         </t>
     </template>
 
     <template id="tax_shelter_subscription_report">
         <t t-call="web.html_container">
             <t t-foreach="docs" t-as="o">
+                <t t-set="lang" t-value="o.partner_id.lang" />
                 <t
                     t-call="l10n_be_cooperator.tax_shelter_report_subscription_document"
                     t-lang="o.partner_id.lang"
```

Comment:
I'm not certain what the improvement here is, but shouldn't `t-lang` on the last line use the set variable instead of getting it from `partner_id`  again?